gallery-so/gallery
# Mobile: Add pinned announcement notification powered by Sanity + Make Mint Flow configurable via Sanity

### Summary of Changes


Announcement
- [x] Add schema for Announcement Notification to sanity
- [x] Display announcement at top of notifications if active
- [x] Announcement data can be refresh via pull-down-to-refresh
- [x] Announcement data is loaded on app start
- [x] Unread dot is shown if announcement has not been seen
- [x] Announcement can be dismissed and state it saved locally

Mint Flow
- [x] Mint Flow can be configured via Sanity
- [x] Supports multiple mint projects at once
- [x] Backwards compatible with ongoing MCHX mint

### Demo or Before/After Pics

Demo on slack

### Edge Cases
- Backwards compatibility with ongoing mint
- sanity data not loading

### Testing Steps

Watch Loom on slack
- turn on announcement (active === true) in Sanity and verify it appears as expected in the app

### Checklist

Please make sure to review and check all of the following:
